keyboards/handwired/tennie/keymaps/default/readme.md

```diff
@@ -0,0 +1,68 @@
+# Default keymap
+
+This keymap is to serve as an example of how you could make a multi-layer keymap.
+
+#### keymap
+
+```
+[base] = LAYOUT_kc(
```

Comment:
QMK advises against using a `LAYOUT_kc`-style macro for a default keymap (though the keymap itself isn't actually using one), because it tends to confuse new users.

This readme should be modified to use the `LAYOUT`-style keycodes that include their `KC_` prefixes.

Reply:
I had that originally, and removed it from the keymaps when it confused someone who was working through the workshop to test it out. So I discovered that first hand! I forgot the layout macro, it's removed now.

Reply:
Could you update the readme, when you get a chance?

Reply:
@OneMoreByte For the record, including a `LAYOUT_kc` macro that automatically prepends `KC_` to all the keycodes it receives (so that `keymap.c` only needs to use the four-character suffix) is fine. But the default keymap or any "reference" keymaps shouldn't use it.

If a user wants to use a `LAYOUT_kc` macro, that's fine.

Reply:
I figured that was the case. I think for the purpose of what I created this for, it really doesn't have a purpose existing though. It will only confuse the people in my workshop.